tests/foreman/cli/test_reporttemplates.py

```diff
@@ -808,3 +818,110 @@ def test_positive_schedule_Entitlements_report(self):
+                          regarding host packages
+
+        :CaseImportance: Medium
+        """
+        # Add subscription to Satellite Tools repo to activation key
+        setup_org_for_a_rh_repo(
+            {
+                'product': PRDS['rhel'],
+                'repository-set': REPOSET['rhst7'],
+                'repository': REPOS['rhst7']['name'],
+                'organization-id': self.setup_org['id'],
+                'content-view-id': self.setup_content_view['id'],
+                'lifecycle-environment-id': self.setup_env['id'],
+                'activationkey-id': self.setup_new_ak['id'],
+            }
+        )
+
+        hosts = []
+        for i in [0, 1]:
+            # Create VM and register content host
+            client = VirtualMachine(distro=DISTRO_RHEL7)
+            client.create()
+            self.addCleanup(vm_cleanup, client)
+            client.install_katello_ca()
+            # Register content host, install katello-agent
+            client.register_contenthost(
+                self.setup_org['label'], self.setup_new_ak['name']
+            )
+            self.assertTrue(client.subscribed)
+            hosts.append(Host.info({'name': client.hostname}))
+            client.enable_repo(REPOS['rhst7']['id'])
+            client.install_katello_agent()
+        hosts.sort(key=lambda host: host['name'])
+
+        host1 = hosts[0]
+        host2 = hosts[1]
+        Host.package_install({'host-id': host1['id'], 'packages': FAKE_0_CUSTOM_PACKAGE_NAME})
+        Host.package_install({'host-id': host1['id'], 'packages': FAKE_1_CUSTOM_PACKAGE})
+        Host.package_install({'host-id': host2['id'], 'packages': FAKE_2_CUSTOM_PACKAGE})
+
+        result = ReportTemplate.generate({'name': 'Host - compare content hosts packages',
+                                         'inputs': f"Host 1 = {host1['name']}, "
+                                          f"Host 2 = {host2['name']}"})
+        result.remove('')
+
+        self.assertGreater(len(result), 1)
+        self.assertEquals(f'Package,{host1["name"]},{host2["name"]},Architecture,Status',
+                          result[0])
+        items = [item.split(',') for item in result[1:]]
+        self.assertGreater(len(items), 0)
+        for item in items:
+            self.assertEquals(len(item), 5)
+            self.assertGreater(len(item[0]), 0)
+            assert((item[1] == '-' and item[0] in item[2])
+                   or (item[0] in item[1] and item[2] == '-')
+                   or (item[0] in item[1] and item[0] in item[2]))
+            self.assertIn(item[3], ['x86_64', 'i686', 's390x', 'ppc64', 'ppc64le', 'noarch'])
+            assert(item[4] == 'same version'
+                   or item[4] == f'{host1["name"]} only'
+                   or item[4] == f'{host2["name"]} only'
+                   or item[4] == f'lower in {host1["name"]}'
+                   or item[4] == f'greater in {host1["name"]}')
```

Comment:
That would be more readable if you use `in` (and possibly assign list of valid values to some variable):
```python
            assert(item[4] in ('same version', f'{host1["name"]} only', f'{host2["name"]} only', f'lower in {host1["name"]}', f'greater in {host1["name"]}'))
```